index.js

```diff
@@ -10,7 +10,7 @@ import {Validator} from "@cfworker/json-schema";
  * Do early composer.json schema validation to avoid unnecessary ramp-ups of jobs which may fail
  * due to an incompatible composer.json.
  */
-if (fs.existsSync('composer.json')) {
+if (fs.existsSync('composer.json') && fs.existsSync('/action/composer.schema.json')) {
```

Comment:
This is kinda unrelated but it always bit me when running this locally due to the absolute path requirement.
I can drop this again.